pkg/operator/config.go

```diff
@@ -83,6 +90,17 @@ func getProviderControllerFromImages(platform configv1.PlatformType, images Imag
 	}
 }
 
+// When the platforme type is BareMetal, MAO is also responsible for deploying
+// the baremetal-operator pod
+func getAdditionalProviderImages(platform configv1.PlatformType, images Images) (bool, string, error) {
```

Comment:
Should this return a list of images? We're going to have others besides the operator.